schemas/v2/index.json

```diff
@@ -399,6 +405,66 @@
             ],
             "additionalProperties": false
         },
+        "GerritConfig": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "const": "gerrit",
+                    "description": "Gerrit Configuration"
+                },
+                "url": {
+                    "type": "string",
+                    "format": "url",
+                    "description": "The URL of the Gerrit host.",
+                    "examples": [
+                        "https://gerrit.example.com"
+                    ],
+                    "pattern": "^https?:\\/\\/[^\\s/$.?#].[^\\s]*$"
+                },
+                "limit":{
```

Comment:
Testing purposes mostly atm. I'm using https://review.opendev.org/ for testing since I don't have a publicly available gerrit instance for testing. I don't want to clone their whole org as this might raise some scrutiny.

Reply:
Makes sense - once we have glob pattern support, this will be less of a issue

Reply:
Also that's useful info about https://review.opendev.org/ - I now see what you mean about paths being used to organize projects.